### DISCUSSION TITLE
Closed issues

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#5304] [General]
We have a bit of a strange situation. There are many closed cases in the milestones that have not yet been released.
I suggest moving them to the already closed 1.6.0.
No objections?

**Comment by whalley** [comment]
Is this the case with issues that were closed but did not required any changes? duplicates? Or have you found some that had code changes but were never tagged to a milestone

**Reply by vomikan** [reply]
It is likely that no work was carried out on such cases, since they remained in the raging milestones.
